spec/services/locale_normalizer_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+describe LocaleNormalizer do
+  describe ".normalize_to_i18n" do
+    it "matches input locales to i18n locales" do
+      expect(described_class.normalize_to_i18n("en-GB")).to eq("en_GB")
+      expect(described_class.normalize_to_i18n("en")).to eq("en")
+      expect(described_class.normalize_to_i18n("zh")).to eq("zh_CN")
+      expect(described_class.normalize_to_i18n("tr")).to eq("tr_TR")
+    end
+
+    it "converts dashes to underscores" do
+      expect(described_class.normalize_to_i18n("a-b")).to eq("a_b")
+    end
+  end
+
+  describe "#is_same?" do
```

Comment:
If you do leave the the `.to_s`-es in `is_same?`, it'd be good to add coverage demonstrating why they're there.